Dockerfile.runner

```diff
@@ -37,7 +37,6 @@ COPY include-bird /etc/dpkg/dpkg.cfg.d/
 
 RUN apt-get update \
     && apt-get install -y software-properties-common \
-    && add-apt-repository -y ppa:smoser/swtpm \
```

Comment:
From 22.04, we need not add this repository.

https://gist.github.com/smoser/ef279e77b068cc081d2f63c10f809db0
> swtpm is available in Jammy (Ubuntu 22.04). ([ITP #941199](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=941199), [debian](https://tracker.debian.org/pkg/swtpm), [ubuntu](https://launchpad.net/ubuntu/+source/swtpm))